engine/src/main/resources/assets/schemas/schemaPlayerSpawnConfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Settings for different ships, classifying the hull and items/money given to the player on spawn.",
+  "type": "object",
```

Comment:
This doesn't match existing spawn configs like https://github.com/MovingBlocks/DestinationSol/blob/develop/modules/core/assets/configs/playerSpawnConfig.json unless i am mistaken (which may be the case)

Reply:
Fixed by changing to "propertyNames" and "additionalProperties", it now matches the spawn configs properly